code/nuxeo-admin-center-jsf/src/main/java/org/nuxeo/connect/client/jsf/AppCenterViewsManager.java

```diff
@@ -641,7 +644,7 @@ public void resetStudioSnapshotValidationStatus() {
     public void setDevMode(boolean value) {
         String feedbackCompId = "changeDevModeForm";
         ConfigurationGenerator conf = setupWizardAction.getConfigurationGenerator();
-        boolean configurable = conf.isConfigurable();
+        boolean configurable = conf.getConfigurationHolder().isLoaded();
```

Comment:
I'm not sure that matches the original use case (static war management): maybe the whole block could be removed here (but could stay as is anyway, that's a detail)

Reply:
Probably not, I'm sure it matches the previous behavior in case the failure is at the beginning of `ConfigurationGenerator#loadConfiguration method`.

The block can probably be removed indeed, but I'm not confident to do it as it could break at runtime.